src/config/compiler.rs

```diff
@@ -82,3 +84,139 @@ pub(super) fn expand_macros(config: &mut Config) -> Result<(), Vec<String>> {
         Ok(())
     }
 }
+
+/// Expand trailing `*` wildcards in input lists
+fn expand_wildcards(config: &mut Config) {
```

Comment:
Would this (and `fn expand_macros` it appears) make more sense to be part of `impl Config` since the only data used comes from the single parameter?

Reply:
The intent is that this is only callable as part of the process of compiling a `ConfigBuilder` into a `Config` (i.e. private to this module). That could be much clearer, and probably should be. The original refactor to force a single path for building `Config`s wasn't taken quite as far as it could have been and this is one place it shows. I'll look into making this a bit clearer.